newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
The other instances of initializing the logger were already moved in 485f99293c280fd6ac99af653736a04804f794d7, so this is the last one.